src/libstore/globals.hh

```diff
@@ -975,6 +975,19 @@ public:
           resolves to a different location from that of the build machine. You
           can enable this setting if you are sure you're not going to do that.
         )"};
+
+    Setting<bool> useXDGBaseDirectories{
+        this, false, "use-xdg-base-directories",
+        R"(
+          If set to `true`, Nix will conform to the [XDG base directory specification]
+          for files in $HOME.
+
+          Note that this changes the location of some well-known symlinks that Nix
+          creates, which might cause some breakages.
```

Comment:
Please pardon me for my nit:
"breakages" -> "breakage"

As I understand it "breakage" is a collective noun, meaning "breakages" would imply multiple collections or sets when I think it is conventional to have one.